src/main/java/com/github/maracas/roseau/Roseau.java

```diff
@@ -62,6 +63,10 @@ final class Roseau implements Callable<Integer>  {
 			description = "Command returns an error when there are breaking changes detected",
 			defaultValue = "false")
 	private boolean failMode;
+	@CommandLine.Option(names = "--format",
+			description="Format of the report; defaults to csv",
+			defaultValue="csv")
+	private String format;
```

Comment:
Instead of an untyped `String`, you should be able to use an enum there (cf. https://picocli.info/#_enum_types). This way, the `switch` below is guaranteed to cover all valid cases.